tests/messages/test_axisarray.py

```diff
@@ -121,19 +126,22 @@ def test_sel():
         data, dims=["dim0"], axes=dict(dim0=AxisArray.Axis(gain=gain, offset=offset))
     )
 
-    aa_sl = aa.sel(dim0=slice(None, -10.75, 1.5))  # slice based on axis info
-    aa_idx = aa.isel(dim0=-1) # index slice of last index
+    aa.sel(dim0=slice(None, -10.75, 1.5))  # slice based on axis info
```

Comment:
`aa.sel(...)` returns something. The better fix here would be to assert something on the return rather than to discard it.

Reply:
Added an assert to check contents of `aa.sel(...)`.